client/verta/.pylintrc

```diff
@@ -1,331 +1,323 @@
+#typealias-rgx=
 
-# Ignore imports when computing similarities.
-ignore-imports=no
+# Regular expression matching correct type variable names. If left empty, type
+# variable names will be checked with the set naming style.
+#typevar-rgx=
 
-# Minimum lines number of a similarity.
-min-similarity-lines=4
+# Naming style matching correct variable names.
+variable-naming-style=snake_case
 
+# Regular expression matching correct variable names. Overrides variable-
+# naming-style. If left empty, variable names will be checked with the set
+# naming style.
+#variable-rgx=
 
-[TYPECHECK]
 
-# List of decorators that produce context managers, such as
-# contextlib.contextmanager. Add to this list to register other decorators that
-# produce valid context managers.
-contextmanager-decorators=contextlib.contextmanager
+[CLASSES]
 
-# List of members which are set dynamically and missed by pylint inference
-# system, and so shouldn't trigger E1101 when accessed. Python regular
-# expressions are accepted.
-generated-members=requests.codes.*
+# Warn about protected attribute access inside special methods
+check-protected-access-in-special-methods=no
 
-# Tells whether missing members accessed in mixin class should be ignored. A
-# mixin class is detected if its name ends with "mixin" (case insensitive).
-ignore-mixin-members=yes
+# List of method names used to declare (i.e. assign) instance attributes.
+defining-attr-methods=__init__,
+                      __new__,
+                      setUp,
+                      asyncSetUp,
+                      __post_init__
 
-# This flag controls whether pylint should warn about no-member and similar
-# checks whenever an opaque object is returned when inferring. The inference
-# can return multiple potential results while evaluating a Python object, but
-# some branches might not be evaluated, which results in partial inference. In
-# that case, it might be useful to still emit no-member and other checks for
-# the rest of the inferred objects.
-ignore-on-opaque-inference=yes
+# List of member names, which should be excluded from the protected access
+# warning.
+exclude-protected=_asdict,_fields,_replace,_source,_make,os._exit
 
-# List of class names for which member attributes should not be checked (useful
-# for classes with dynamically set attributes). This supports the use of
-# qualified names.
-ignored-classes=optparse.Values,
-                thread._local,
-                _thread._local,
+# List of valid names for the first argument in a class method.
+valid-classmethod-first-arg=cls
 
-# List of module names for which member attributes should not be checked
-# (useful for modules/projects where namespaces are manipulated during runtime
-# and thus existing member attributes cannot be deduced by static analysis. It
-# supports qualified module names, as well as Unix pattern matching.
-ignored-modules=verta._protos.*,
-                google.protobuf.struct_pb2,
```

Comment:
Moved to https://github.com/VertaAI/modeldb/pull/3834/files#diff-34a29357b054cae79e8b110c487275043f298eac35574ed53e7fddb461dd220aR70-R71